test/over_react/util/map_util_test.dart

```diff
@@ -122,6 +122,120 @@ main() {
       });
     });
 
+    group('forwardUnconsumedProps() returns a copy of the specified props', () {
```

Comment:
This description needs to be updated; this function does not return anything

Reply:
Yep - should have caught that one. Thank you!